### DISCUSSION TITLE
请问一下，拼音输入中有些奇怪的编码在哪里设置？

### DISCUSSION THREAD

**Discussion by Hhsy083212** [fcitx5-android/fcitx5-android#612] [Q&A | 问答] [answered]
![Screenshot_2024-11-10-04-22-19-140_com miui notes](https://github.com/user-attachments/assets/4bdff857-e3dd-4fef-adfe-384cf21d9ff1)

![Screenshot_2024-11-10-04-33-23-494_com miui notes](https://github.com/user-attachments/assets/9547d546-cd8e-44a1-bbd7-23bc4f8f84e0)

如图所示，默认的拼音码表，第五个带注释的是怎么回事？怎样才能消除呢？

**Comment by Hhsy083212** [comment]
顺便问一下各位大神，目前默认的拼音码表会中英文混输，请问怎样才能设置为不显示英文呢？
求解答，谢谢。

**Comment by rocka** [comment, marked as answer]
> 默认的拼音码表，第五个带注释的是怎么回事？怎样才能消除呢？

是笔划输入功能。 hspnz 分别代表 横竖撇捺折， hh → 横横 → 二。目前还不支持单独关闭。

> 目前默认的拼音码表会中英文混输，请问怎样才能设置为不显示英文呢？

拼音设置，关闭“启用拼写检查”。

另外需要说明一下， fcitx5 的“拼音”和“码表”是两个独立的引擎，前者是常用的拼音/双拼输入法实现，后者是类似五笔的形码输入法实现，建议不要把这两个概念混淆，以免产生误解。

**Reply by Hhsy083212** [reply]
😁😁谢谢大佬回复，已经设置好了，越用越趁手。
